src/UserNSRunner.jl

```diff
@@ -62,6 +62,19 @@ function UserNSRunner(workspace_root::String;
         push!(workspaces, ccache_dir() => "/root/.ccache")
     end
 
+    # If we are on a system that uses `/etc/resolv.conf` as a resolver
+    # configuration file (mainly *BSD and Linux), check if a nameserver is
+    # specified and if yes, mount the existing `/etc/resolv.conf` to make use
+    # of system-specific nameserver settings
+    if (Sys.isbsd() || Sys.islinux()) && isfile("/etc/resolv.conf")
+        for line in eachline("/etc/resolv.conf")
+            if startswith(line, "nameserver")
+              push!(workspaces, "/etc/resolv.conf" => "/etc/resolv.conf")
+              break
+            end
```

Comment:
So this will push an entry onto `workspaces` for _each_ `line` that starts with `nameserver`; not quite what we want.  :)

Try something more like `if any(startswith(line, "nameserver") for line in eachline("/etc/resolv.conf"))`.

Reply:
I don't think it will do that, does it? The `break` will exit the loop over each lines once the first matching line is found, thus executing `push!` at most once.

Anyhow, since my approach is obviously not as easy to understand, I used your approach in be21700a584cc088bda0b09bf7e9d419b6a30277.

Reply:
Wow, my eyes totally missed the `break`.  You're right, of course.